assets/stylesheets/main.css

```
@@ -0,0 +1,186 @@
+/*
+  ========================================
+  Border Box  Layout
+  ========================================
+*/
+
+*,
+*::before,
+*::after {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+/* General CSS */
+
+.flex {
+  display: flex;
+  align-items: center;
+}
+
+.flex-2 {
+  display: flex;
+  justify-content: center;
+}
```

group selectors:
```suggestion
.flex, .flex-2 {
  display: flex;
  align-items: center;
}
```
or create a new class:
```suggestion
.new_class_name {
  display: flex;
  align-items: center;
}
```